db/sql/create-timeEntry-and-timePeriodType-table.sql

```diff
@@ -0,0 +1,32 @@
+
+CREATE TABLE timecard.time_period_type (
+    id VARCHAR(36)  PRIMARY KEY,
+    tenant_id VARCHAR(36) NOT NULL,
+    name VARCHAR(50) NOT NULL
+);
+
+INSERT INTO timecard.time_period_type (id,tenant_id,"name") VALUES
+	 ('00000000-0000-0000-0000-000000000001','00000000-0000-0000-0000-000000000000','Shift'),
+	 ('00000000-0000-0000-0000-000000000002','00000000-0000-0000-0000-000000000000','Scheduled rest day'),
+	 ('00000000-0000-0000-0000-000000000003','00000000-0000-0000-0000-000000000000','Non-working day'),
+	 ('00000000-0000-0000-0000-000000000004','00000000-0000-0000-0000-000000000000','On call'),
+	 ('00000000-0000-0000-0000-000000000005','00000000-0000-0000-0000-000000000000','Absence'),
+	 ('00000000-0000-0000-0000-000000000006','00000000-0000-0000-0000-000000000000','Training'),
+	 ('00000000-0000-0000-0000-000000000007','00000000-0000-0000-0000-000000000000','Overtime');
+
+CREATE TABLE timecard.time_entry (
+    id VARCHAR(36)  PRIMARY KEY,
+    tenant_id VARCHAR(36) NOT NULL,
+    version INT DEFAULT NULL,
```

Comment:
I’m not sure what some of these columns are for, `version` being one of them.

Probably a question for @wweatherill-homeoffice and @dawatkin3 - would it be acceptable to leave them out of the schema for now and add them when we get around to building the features that use them?

I think we’d benefit from starting with the database schema as simple as it’s able to be and allowing it to grow incrementally.

Reply:
@stephenkaye-ho also may have an opinion on this

Reply:
I will wait for Will/David to Comment on this